docs/developers/overview.md

```diff
@@ -18,7 +18,7 @@ license: |
 
 # Celeborn Architecture
 
-This article introduces high level Apache Celeborn(Incubating) Architecture. For more detailed description of each module/process,
+This article introduces high level Apache Celeborn™(Incubating) Architecture. For more detailed description of each module/process,
```

Comment:
Miss the `Apache Celeborn(Incubating) solves the problems by reorganizing shuffle data in a more efficient way` in `overview.md`.